packages/react-next/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -2,257 +2,221 @@ import * as React from 'react';
 import { FocusZone, FocusZoneDirection, IFocusZone } from '@fluentui/react-focus';
 import { IKeytipProps } from '../../Keytip';
 import {
-  initializeComponentRef,
   classNamesFunction,
   divProperties,
   elementContains,
-  focusFirstChild,
   getNativeProps,
   warnMutuallyExclusive,
+  focusFirstChild,
 } from '../../Utilities';
 import { IProcessedStyleSet } from '../../Styling';
 import { KeytipManager } from 'office-ui-fabric-react/lib/utilities/keytips/KeytipManager';
 import {
-  IOverflowSet,
   IOverflowSetItemProps,
   IOverflowSetProps,
   IOverflowSetStyles,
   IOverflowSetStyleProps,
+  IOverflowSet,
 } from './OverflowSet.types';
 
 const getClassNames = classNamesFunction<IOverflowSetStyleProps, IOverflowSetStyles>();
 const COMPONENT_NAME = 'OverflowSet';
-
-export class OverflowSetBase extends React.Component<IOverflowSetProps, {}> implements IOverflowSet {
-  private _focusZone = React.createRef<IFocusZone>();
-  private _persistedKeytips: { [uniqueID: string]: IKeytipProps } = {};
-  private _keytipManager: KeytipManager = KeytipManager.getInstance();
-  private _divContainer = React.createRef<HTMLDivElement>();
-  private _classNames: IProcessedStyleSet<IOverflowSetStyles>;
-
-  constructor(props: IOverflowSetProps) {
-    super(props);
-
-    initializeComponentRef(this);
-    warnMutuallyExclusive(COMPONENT_NAME, props, {
-      doNotContainWithinFocusZone: 'focusZoneProps',
+let Tag;
+let uniqueComponentProps;
+
```

Comment:
I think @xugao was doing some work related to keytips--does this conflict at all with that? And can/should it be moved to a more general location?

Reply:
Thanks for pointing this out Elizabeth. To answer you, this doesn't conflict with my changes directly. `OverflowSet` is the only component which directly uses `KeytipManger` (instead of `KeytipData`). 
So there is no need to make this code more generic for now. I will take a deeper look on if we can remove `KeytipManager` from `OverflowSet` completely